v22.1/row-level-ttl.md

```diff
@@ -455,6 +461,14 @@ Row-level TTL interacts with [backup and restore](backup-and-restore-overview.ht
 
 - When you [`RESTORE`](restore.html) from a backup, all row-level TTL information associated with the tables being restored (including TTL expiration times) is also restored. Any expired rows in the restored tables are eligible to be [deleted](#when-are-rows-deleted) by the [TTL job](#how-it-works).
 
+## Required Privileges
+
+To add or update Row-Level TTL settings on a table, you must have one of the following:
+
+- Membership to the [`admin`](security-reference/authorization.html#roles) role for the cluster.
```

Comment:
@ecwall @rafiss Do we have an issue for taking a second look at privileges for 22.2? I don't think we want to add more to the `admin` role.